docs/reference/01-scnode-api-spec.rst

```diff
@@ -2413,3 +2413,191 @@ ________
+
+*Creates and signs sidechain transaction for signers or masters certificate submitter key rotation.*
+
+**Parameters**
+
++---------------------+-----------+-----------------------------------------------------------------------+
+| Name                | Type      |            Description                                                |
++=====================+===========+=======================================================================+
+| keyType             |  int      | Key type - 0 for signers key, 1 for masters key. Min = 0. Max = 1     |
++---------------------+-----------+-----------------------------------------------------------------------+
+| keyIndex            |  int      | Index of certificate submitter key                                    |
++---------------------+-----------+-----------------------------------------------------------------------+
+| newKey              |  string   | Value of new key                                                      |
++---------------------+-----------+-----------------------------------------------------------------------+
+| signingKeySignature |  string   | Signing key signature                                                 |
++---------------------+-----------+-----------------------------------------------------------------------+
+| masterKeySignature  |  string   | Master key signature                                                  |
++---------------------+-----------+-----------------------------------------------------------------------+
+| newKeySignature     |  string   | New key signature (if key type 0, then new signers key signature;     |
+|                     |           | if key type 1, then master key signature). Min = 0. Max = 1.          |
++---------------------+-----------+-----------------------------------------------------------------------+
+| format              |  boolean  | Optional field - true if format, false if non format                  |
++---------------------+-----------+-----------------------------------------------------------------------+
+| automaticSend       |  boolean  | Optional field - true if automatic, false if not automatic            |
++---------------------+-----------+-----------------------------------------------------------------------+
+| fee                 |  int      | Optional field for transaction fee                                    |
++---------------------+-----------+-----------------------------------------------------------------------+
+
+**Example request**:
+
+.. tabs::
+
+   .. tab:: Bash
+
+      curl -X POST \"http://127.0.0.1:9085/transaction/createKeyRotationTransaction\" -H \"accept: application/json\" -d \"{\\\"keyType\\\": 0, \\\"keyIndex\\\": 3, \\\"newKey\\\":\\\"string\\\, \\\"signingKeySignature\\\":\\\"string\\\, \\\"masterKeySignature\\\":\\\"string\\\, \\\"newKeySignature\\\":\\\"string\\\"}\"
+
+
+**Example response**:
+
+   .. sourcecode:: http
+
+      {
+         "result":{
+
```

Comment:
The result should be empty. is it correct?

Reply:
Transaction is failing, debugging the reason

Reply:
Wait for certificate in mc mempool...
Wait for certificate in mc mempool...
[INFO] [akkaDeadLetter][12/09/2022 11:38:20.998] [2-Hop-akka.actor.default-dispatcher-6] [akka://2-Hop/system/IO-TCP] Message [akka.io.Tcp$Unbind$] from Actor[akka://2-Hop/user/networkController#-800685663] to Actor[akka://2-Hop/system/IO-TCP#950087774] was unhandled. [1] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[INFO] [12/09/2022 11:38:21.015] [Thread-3] [CoordinatedShutdown(akka://2-Hop)] Running CoordinatedShutdown with reason [ActorSystemTerminateReason]
Dec 09, 2022 11:38:21 AM org.glassfish.grizzly.http.server.NetworkListener shutdownNow
INFO: Stopped listener bound to [0.0.0.0:8025]
Dec 09, 2022 11:38:21 AM org.glassfish.tyrus.server.Server stop
INFO: Websocket Server stopped.
double free or corruption (out)